TwoADuodecimalRangeChecker/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace TwoADuodecimalRangeChecker
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
a can be less then b , is a tasks no limitations to the order of input values. One of the numbers can be negative. Please process this case

Reply:
changes done, now works with negative numbers as well + a and b order doesn't affect the behavior of the appliation